hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -175,7 +175,8 @@ public long fetchTotalInsertRecordsWritten() {
     long totalInsertRecordsWritten = 0;
     for (List<HoodieWriteStat> stats : partitionToWriteStats.values()) {
       for (HoodieWriteStat stat : stats) {
-        if (stat.getPrevCommit() != null && stat.getPrevCommit().equalsIgnoreCase("null")) {
+        // It was only possible to determine the number of rows to insert for the first commit before. Currently, this problem is fixed
```

Comment:
nit : no need to explain "Currently, this problem is fixed" :) We can just comment saying "determine the number of rows to insert for the first commit". Once you do that, we can merge this!

Reply:
@n3nash 
OK~ have changed it.

Reply:
Thank you!